src/python/bot/fuzzers/engine.py

```diff
@@ -15,8 +15,13 @@
 
 from builtins import object
 from collections import namedtuple
+import random
+
+from system import environment
 
 _ENGINES = {}
+_TRIAL_PROBABILITY_FUZZ = 0.2
+_TRIAL_PROBABILITY_OTHERS = 0.05
```

Comment:
Might be better to start with 0 first, fix the fuzzing failures first. breaking existing open testcases is very painful, maybe just use USE_NEW_ENGINE_IMPL on clusterfuzz.corp where changes are not visible atm and we will be refiling bugs later ?

Reply:
Sure, I already tested this with some test jobs, but will set to 0 for now to be extra careful.